lib/airbrake.js

```diff
@@ -316,12 +316,21 @@ Airbrake.prototype.addRequestVars = function(request, type, vars) {
 
 Airbrake.prototype.cgiDataVars = function(err) {
   var cgiData = {};
-  Object.keys(process.env).forEach(function(key) {
-    cgiData[key] = process.env[key];
-  });
-
   var self = this;
 
+  if (this.whiteListKeys.length > 0){
+    this.whiteListKeys.forEach(function(key) {
+      cgiData[key] = process.env[key];
+    });
+  } else if (this.blackListKeys.length > 0){
+    Object.keys(process.env).forEach(function(key) {
+      if (self.blackListKeys.indexOf(key) > -1){
+        return;
```

Comment:
What do you think about saying `[Filtered]` similar to [what we already do](https://github.com/airbrake/airbrake-ruby/blob/master/lib/airbrake-ruby/filters/keys_filter.rb#L49) in the Ruby notifier? It's more descriptive and lets the user know that it was filtered, not missed.